packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -64,6 +71,35 @@ export class InsuredBridgeL2Client {
     return this.deposits[depositHash];
   }
 
+  getTokensBridgedTransactionsForL2Token(l2TokenAddress: string) {
+    if (!this.tokensBridgedTransactions[l2TokenAddress]) return [];
```

Comment:
accessor methods. some of these are a bit convoluted due to the data structure. I played with a good number of these and this implementation worked out to be the simplest version, mainly due to the data we need to store and access without adding additional async calls.